dev/partkey.md

```diff
@@ -0,0 +1,165 @@
+---
+numbersections: true
+title: "Algorand Key Specification"
+date: \today
+abstract: >
+    This document specifies the list of keys and their capabilities in Algorand.
+---
+
+Overview
+========
+
+An algorand node uses two kind of cryptographic keys:
+
+ - _participation keys_, a set of keys used for authentication, i.e. identify an 
+    account. Algorand uses a hierarchical (two\-level) identity encryption scheme that 
+    ensures [forward security](https://en.wikipedia.org/wiki/Forward_secrecy), 
+    which will be detailed in next section. 
+
+ - _selection credential_, key used for proving membership of selection. 
+
+A vote is valid only if both the participation keys and the selection
```

Comment:
With the new proposed naming, isn't it "A participation key".

Also, how do you separate the public key part and the private key part (in the naming)?